src/graphics.cpp

```diff
@@ -105,9 +249,35 @@ void GetvScreen(const int A)
             if(vScreen[A].TempY < 0)
                 vScreen[A].TempY += 1;
         }
-        if(p.Mount == 2)
-            pLoc.Height = 128;
     }
+
+    // keep vScreen boundary even on 3DS
+#ifdef __3DS__
+    vScreenX[A] -= std::fmod(vScreenX[A], 2.);
+    vScreenY[A] -= std::fmod(vScreenY[A], 2.);
```

Comment:
Please don't use floating point numbers without zeroes. I mean, using `2.0` or `2.0f` or in the worst case `2.f` is fine, but `2.` is not so good.